views/FanView/FanView.tsx

```diff
@@ -24,6 +24,11 @@ export const FanView: FC<FanViewProps> = ({ route }) => {
     selectDeviceWithId(state, location, nodeId, deviceId),
   ) as Fan;
 
+  const fanTurnedOff =
+    fan.values.V_TEMP === '0' &&
+    fan.values.V_PERCENTAGE === '0' &&
+    fan.values.V_DIRECTION === '0';
```

Comment:
Should be wrapped in useMemo (now when you change input value this variable will be recomputed)